### ISSUE TITLE
[WCAG2ICT] First Public Working Group Note

### ISSUE THREAD

**Issue by daniel-montalvo** [w3c/transitions#562]
# WCAG2ICT

[https://wwww.w3.org/TR/2023/DNOTE-wcag2ict-20230815/](https://wwww.w3.org/TR/2023/DNOTE-wcag2ict-20230815/)

15 August 2023.

## Abstract

This document, “Guidance on Applying WCAG 2.2 to Non-Web Information and Communications Technologies (WCAG2ICT)” describes how the Web Content Accessibility Guidelines (WCAG) 2.2 [WCAG22] and its principles, guidelines, and success criteria can be applied to non-web Information and Communications Technologies (ICT), specifically to non-web documents and software. It provides informative guidance (guidance that is not normative and does not set requirements).

This document is part of a series of technical and educational documents published by the W3C Web Accessibility Initiative (WAI) and available from the WCAG2ICT Overview.

## Status of these document

This section describes the status of this document at the time of its publication. A list of current W3C publications and the latest revision of this technical report can be found in the W3C technical reports index at https://www.w3.org/TR/.

This is a First Public Working Draft of Guidance on Applying WCAG 2.2 to Non-Web Information and Communications Technologies (WCAG2ICT). The intent of this work is to update the existing guidance based on new WCAG 2.1 and 2.2 success criteria.

The current draft includes guidance for WCAG 2.1 success criteria. Guidance on WCAG 2.2 success criteria will be added later this year as WCAG 2.2 becomes a W3C Recommendation. The next draft will also address open issues on WCAG 2.0 criteria.

The group is seeking feedback on the following aspects:

• New added guidance for WCAG 2.1 success criteria and glossary term definitions

• Updates to the closed functionality guidance for new WCAG 2.1 success criteria

The group is already aware of formatting differences in how quoted passages from WCAG are included in comparison to the previous WCAG2ICT Working Group Note.

Currently there are known issues in the quoted content from WCAG and Understanding Intent sections:

• Anchor links from the included WCAG content to sections that do not exist in the WCAG2ICT document don't work

• The words "Note" and "Example" are missing from notes and examples of quoted WCAG passages

The group is currently working on updates to the format, styling, and linking for these passages, and expects to refine these in a future draft.

This document was published by the Accessibility Guidelines Working Group as a Group Draft Note using the Note track.

Group Draft Notes are not endorsed by W3C nor its Members.

This is a draft document and may be updated, replaced or obsoleted by other documents at any time. It is inappropriate to cite this document as other than work in progress.

The 1 August 2017 W3C Patent Policy does not carry any licensing requirements or commitments on this document.

This document is governed by the 12 June 2023 W3C Process Document.

## Decision URL

[https://lists.w3.org/Archives/Public/w3c-wai-gl/2023JulSep/0208.html](https://lists.w3.org/Archives/Public/w3c-wai-gl/2023JulSep/0208.html)

**Comment by plehegar** [comment]
Approved

**Comment by plehegar** [comment]
https://www.w3.org/TR/2023/DNOTE-wcag2ict-20230815/
